build.gradle.kts

```diff
@@ -151,18 +162,20 @@ flyway {
     url = postgresUrl
     user = postgresUser
     password = postgresPassword
+    // The default is true, and Flyway gradle plugin will not be executed in production environment.
+    cleanDisabled = false
 }
```

Comment:
This update PR looks super painful until now 😆

Reply:
Yeah, especially getting Gradle to work (always) takes much time. Dart “pub” is a much better tool, from my point of view.